modules/programs/ssh.nix

```diff
@@ -24,6 +28,15 @@ let
         description = "Specifies port number to connect on remote host.";
       };
 
+      forwardAgent = mkOption {
+        default = null;
+        type = types.nullOr types.bool;
```

Comment:
I think you can simply have `type = types.bool;` here and set the default to `false`. Then have

```nix
    ++ optional cf.forwardAgent "  ForwardAgent yes"
```
below.

Reply:
Does it inherit the settings from the `*` host?

Reply:
That is a fair point! My suggestion would not allow setting a default value in the `*` block. Better to keep it as `types.nullOr types.bool`.